課題提出

keiz1213 opened the pull request:
よろしくおねがいします。

keiz1213 committed:
課題提出

keiz1213 committed:
不要と思われるファイルをgitignoreに追記

keiz1213 committed:
タイポ修正

yuuu commented on memo-app/src/components/MemoDetail.vue:
```diff
@@ -0,0 +1,65 @@
+  methods: {
+    fetchData: function () {
+      const filterdMemos = this.memos.filter(memo => {
+        return memo.id === this.$route.params.memoId
```
アロー関数で `return` のみを実行する場合、 `{}` と `return` は省略できますね 👍

keiz1213 replied:
確かにです💡こちら修正しました。

yuuu commented on memo-app/src/router/index.js:
```diff
@@ -0,0 +1,32 @@
+
+const routes = [
+  {
+    path: '/',
```
`<vue-router>` にて `v-show !== '/'` としているので、ここの指定は不要なのでは？と思いました。ないとエラーになりますか？？

yuuu replied:
`router.push('/')` している箇所があるので必要かもしれませんね 🤔

keiz1213 replied:
こちらを削除して手元で試してみたところ、確かにエラーが出ずに正常に動きました！なので削除して修正しています😄

yuuu commented on memo-app/src/components/MemoList.vue:
```diff
@@ -0,0 +1,25 @@
+  <div class="memo-list">
+    <h1>メモ一覧</h1>
+    <div v-for="memo in memos" :key="memo.id">
+      <router-link :to="{ path: '/memos/' + memo.id}">{{ memo.content.split(/\r\n|\r|\n/)[0] }}</router-link>
```
pathの部分は `` `/memos${memo.id}` `` としても良いですね

keiz1213 replied:
確かにテンプレートリテラルで書いたほうが見やすいし短くなってスッキリしました💡ありがとうございます！

keiz1213 committed:
指摘事項修正

1.アロー関数を正しい構文で修正
2.routesから'/'と'MemoBoard'とのマッピングを削除
3.router-linkのtoプロパティへ渡すpathの記述方法を修正
